extensions/websockets/server/deployment/src/main/java/io/quarkus/websockets/deployment/ServerWebSocketProcessor.java

```diff
@@ -110,10 +110,11 @@ public FilterBuildItem deploy(final CombinedIndexBuildItem indexBuildItem,
         final IndexView index = indexBuildItem.getIndex();
         WebsocketClientProcessor.registerCodersForReflection(reflection, index.getAnnotations(SERVER_ENDPOINT));
 
+        int priority = 1 + FilterBuildItem.AUTHORIZATION;
```

Comment:
This is (old) WebSocket so it shouldn't matter and I don't know anything about it, but for record, I think it would be better if this was the opposite - `1-FilterBuildItem.AUTHORIZATION` because this priority is multiplied by `-1` and it means that this handler will run before the authorization handler. If they have some HTTP permissions, they will not run before WS handler.

Reply:
Feel free to open a PR in that case

Reply:
I'd have to analyze why it worked this way before (why was the priority same), how the legacy WS works etc. I don't think I'll do that. I just tried to provide information, I have no problem with keeping what you have.